hpy/tools/autogen/debug.py

```diff
@@ -47,7 +47,7 @@ def generate(self):
         w('{')
         for var in self.api.variables:
             name = var.name
-            w(f'    dctx->{name} = DHPy_open(dctx, uctx->{name});')
+            w(f'    dctx->{name} = DHPy_open_context_constant(dctx, uctx->{name});')
```

Comment:
I really like the idea, but `context_constant` is a bit weird IMHO. What about `DHPy_open_immortal`?